deploy/common/ci-values.yaml

```diff
@@ -43,3 +43,8 @@ scannerV4:
       limits:
         memory: "4Gi"
         cpu: "2000m"
+
+customize:
+  scanner-v4-matcher:
+    envVars:
+      SCANNER_V4_MATCHER_READINESS: vulnerability
```

Comment:
Just curious: I think this'll take lets say 15 - 20 minutes. Is that ok? Are our tests willing to wait that long?